scripts/update/update-grafana-dashboard.sh

```
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+UPSTREAM_DASHBOARD_URL=https://raw.githubusercontent.com/GreptimeTeam/greptimedb/refs/heads/main/grafana/dashboards/cluster/dashboard.json
+
+update_chart_version() {
+  # Extract the version and increment the last digit by 1.
+  VERSION=$(cat charts/greptimedb-cluster/Chart.yaml | grep -m 1 'version:' | awk '{print $2}')
+  VERSION_PARTS=(${VERSION//./ })
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)